src/server/helpers/env.ts

```diff
@@ -13,9 +13,10 @@ function getFromEnv_(
     return process.env[key];
   }
   if (isRequired) {
-    throw new Error(`ENV undefined key: ${key}.`);
+    console.error(new Error(`ENV undefined key: ${key}.`));
```

Comment:
Throwing werkt onprettig

Reply:
dit wordt niet opgepakt door de azure logging? of is dat hier geen probleem?